spec/services/authentication/authenticator_spec.rb

```diff
@@ -717,19 +975,31 @@
         ).to be(true)
       end
 
-      it "does not update their twitter_username if the user is suspended" do
+      it "increments identity.errors if any errors occur in the transaction" do
```

Comment:
Quick, non-blocking suggestion (same as above!): Let's aggregate the failures here since we have multiple expectations within this single block:
```suggestion
      it "increments identity.errors if any errors occur in the transaction", :aggregate_failures do
```